test_scripts/Defects/5_1/980_SDL_responds_GENERIC_ERROR_instead_of_INVALID_DATA_when_soft_button_has_Type_is_Image_or_Both_and_Text_is_whitespace_or_\t_or_\n_or_empty.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/980
+-- Description: [APIs] AlertManeuver: SDL responds GENERIC_ERROR instead of INVALID_DATA when soft button has Type is
+-- Image or Both and Text is whitespace or \t or \n or empty
+-- Precondition:
+-- 1) AlertManeuver is allowed by policy for "default" group.
+-- 2) SDL and HMI are started.
+-- 3) App is connected and at the FULL.
+-- In case:
+-- 1) Send AlertManeuver sure that soft button has Type is Image or Both and Text is whitespace or \t or \n or empty.
+-- Expected result:
+-- 1) SDL must respond with resultCode "INVALID_DATA" and success:"false".
+-- Actual result:
```

Comment:
Please remove the actual result

Reply:
@GetmanetsIrina You can find the updates in the commit dac0e07